api/src/main/java/com/eleks/academy/pharmagator/dataproviders/PharmacyANCDataProvider.java

```diff
@@ -0,0 +1,29 @@
+package com.eleks.academy.pharmagator.dataproviders;
+
+
+import com.eleks.academy.pharmagator.dataproviders.dto.MedicineDto;
+import com.eleks.academy.pharmagator.dataproviders.dto.anc.ANCMedicineDto;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.reactive.function.client.WebClient;
+import java.util.stream.Stream;
+
+public class PharmacyANCDataProvider implements DataProvider {
+
+    //private final WebClient ancClient;
+
+    @Value("${pharmagator.data-providers.apteka-anc.category-fetch-url}")
+    private String categoriesFetchUrl;
+
+    @Override
+    public Stream<MedicineDto> loadData() {
+        return null;
```

Comment:
NOPE!
Never `null`. Never.
If you wanna mock an empty response use `Stream.empty()`, `Optional.empty()`, `List.of()`, `Maybe.empty()`, but not `null`. I may want to consume your service and I never expect `Stream<>` response to be `null`